roles/observe_chargeback/README.md

```diff
@@ -0,0 +1,56 @@
+Role Name
```

Comment:
```suggestion
observe_changeback
```

However, I suggest changing the name of the role to telemetry_chargeback, to be in line with the other tests roles that are used for openstack-k8s-operators/telemetry-operator-deployed components